packages/components/src/Progress/Progress.component.js

```diff
@@ -24,14 +24,15 @@ function Progress({ id, percent, tooltip, infinite, contained }) {
 		{
 			[theme.hidden]: normalizedPercent === 0,
 			[theme.fixed]: !contained,
+			[theme.infinite]: !!infinite,
 		}
 	);
-	const progressionClassNames = classNames(
-		theme['progress-percent'],
-		{ [theme.infinite]: !!infinite }
-	);
 
-	let progress = <div style={style} className={progressionClassNames} />;
+	let progress = (
+		<div style={style} className={classNames(theme['progress-percent'])}>
```

Comment:
You don't need classNames, you only compose 1 class
`className={theme['progress-percent']}`